projects/angular/src/progress/progress-bars/_progress-bars.clarity.scss

```diff
@@ -561,6 +561,10 @@
       // by default, positioned at the topmost of its card-block container
       margin: 0;
       margin-top: -1 * card-variables.$card-children-padding-vertical;
+      @if variables.$clr-use-custom-properties == true {
```

Comment:
I think I am also doing something similar in the combobox. I was thinking what will happen if we remove the "exports" mixin, as we talked that we may after moving to @use. Won't all these @ifs we put stop working?

Reply:
Pls ignore for the time being.